### PR TITLE
Add Geyser tag

### PR DESCRIPTION
**Pull request by TwoLeggedCat**
Iffy on using the avatar URL, but it does make it auto-update if they change it ¯\\\_(ツ)_/¯

CC @Camotoy for comments (is there a more permanent downloads link I should use?)

### CODE REVIEW

File: GeyserMC.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+"embed": {
+
+    "description": "Geyser is a Bukkit plugin that allows Bedrock Edition players to connect to Java Edition servers.",
```
Comment (Camotoy) [inline GeyserMC.json:4]: Geyser isn't just a Spigot plugin - it also works on Velocity, BungeeCord, Sponge, Fabric, and as a standalone instance. Also, we typically use "Spigot" wording for the Bukkit platform so people don't try to use CraftBukkit.

**Comment by Camotoy** [comment]
Looks fine otherwise.

File: GeyserMC.json (hunk 1 above)
Comment (TwoLeggedCat) [reply GeyserMC.json:4]: Mmmm, good point. That wouldn't effect most of Syscraft anyway but I'll change it.

Comment (TwoLeggedCat) [reply GeyserMC.json]: done

**Pull request merged**